.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: node_js
+node_js:
+    - 8
```

Comment:
8 is EOL in about 3 months. Why not use current?

Reply:
Sure!